src/content/apis/animals/cat-facts.md

```diff
@@ -5,6 +5,7 @@ categories:
   - animals
 features:
   https: true
+  payment: Upgradable
```

Comment:
I am completely trusting you with the research you have done on your end.

However can you make the values all lowercase i.e. `upgradable`, `free` or `paid`.
Please make this change in all of the files 🙏